app/src/reducers/map.js

```diff
@@ -24,7 +23,7 @@ import _ from 'lodash';
 import { DEFAULT_VESSEL_COLOR } from 'constants';
 
 const initialState = {
-  active_basemap: 'satellite',
+  activeBasemap: 'satellite',
```

Comment:
For the sake of sanity; let's avoid duplicating source of truth and remove this since this is already in the default workspace.